asv_bench/benchmarks/indexing.py

```diff
@@ -11,95 +11,110 @@
 class NumericSeriesIndexing(object):
 
     goal_time = 0.2
-    params = [Int64Index, Float64Index]
-    param = ['index']
+    params = [
+        (Int64Index, Float64Index),
+        ('unique_monotonic_inc', 'nonunique_monotonic_inc'),
+    ]
+    param_names = ['index dtype', 'index structure']
```

Comment:
did you actually try this? I am shocked if this works when it has spaces in the name, or maybe the names are just mapped by postion. in any event these need underscores

Reply:
Yeah I tried this and you can see it working in this sample output from the asv run:
```
[ 97.63%] ··· indexing.NumericSeriesIndexing.time_loc_list_like                                                     ok
[ 97.63%] ··· ========================================== ========================= ============
                             index dtype                      index structure                  
              ------------------------------------------ ------------------------- ------------
                pandas.core.indexes.numeric.Int64Index      unique_monotonic_inc     379±2μs   
                pandas.core.indexes.numeric.Int64Index    nonunique_monotonic_inc   69.3±0.5ms 
               pandas.core.indexes.numeric.Float64Index     unique_monotonic_inc     170±1ms   
               pandas.core.indexes.numeric.Float64Index   nonunique_monotonic_inc   65.7±0.3ms 
              ========================================== ========================= ============
```

I've added underscores in commit `4ad3006`.